core/sbt-cloudflow/src/main/scala/cloudflow/sbt/CloudflowKeys.scala

```diff
@@ -72,8 +72,8 @@ trait CloudflowTaskKeys {
   private[sbt] val allProjectsWithStreamletScannerPlugin =
     taskKey[Seq[ProjectReference]]("All projects that use the streamlet scanner plugin.")
 
-  private[sbt] val allProjectDependenciesWithCloudflowBasePlugin =
-    taskKey[Seq[ProjectReference]]("All projects that are dependencies of the current project and use the CloudflowBasePlugin.")
+  private[sbt] val allProjectsWithCloudflowBasePlugin =
```

Comment:
reverted back to original task name and description, because using the original filter also changes taskKey semantics back.